### ISSUE TITLE
Add the ability to import RDi configurations

### ISSUE THREAD

**Issue by ChrisHird** [codefori/vscode-ibmi#1295]
Would like to be able to import compiler settings etc from RDi. Today we have to build the commands manually which can be a pain why trying to migrate 7 development environments. Tried to figure out the RDi configuration layout and see if I could build something to do the import but need more information on how IBM exports and imports the configurations?

Just an idea as I am migrating from RDi to VSCode.

**Comment by worksofliam** [comment]
@ChrisHird Any chance you can share an example of those settings or the schema for them?

Thanks

**Comment by ChrisHird** [comment]
Yes I can add to the issue, I have also asked for help on the possible format or how it is decoded in RDi. Will add what I can as I find out.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by worksofliam** [comment]
@ChrisHird Any luck getting an example configuration for us?

**Comment by ChrisHird** [comment]
Sorry Liam I am travelling, I will copy Charlie and get him to create one.

Chris

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by worksofliam** [comment]
@ChrisHird Perfect, thanks.

Please note attachments do not come through via email and will need to be attached in this issue on GitHub.

**Comment by ChrisHird** [comment]
[conf.zip](https://github.com/halcyon-tech/vscode-ibmi/files/11568133/conf.zip)

**Comment by alanseiden** [comment]
@ChrisHird Thank you. We found that the inner .cfg files and so forth are really additional .zip files and can be renamed .zip and uncompressed. Then we figure out which files we need.

**Comment by alanseiden** [comment]
A community member suggests looking at iSphere RDi plugin for a way to export sections of the .cfg files.

**Comment by worksofliam** [comment]
Not something we're going to implement any time soon. PRs are welcome.
